src/lib/Zikula/Bundle/CoreBundle/EventListener/ThemeListener.php

```diff
@@ -95,23 +97,29 @@ public function setDefaultPageAssets(GetResponseEvent $event)
             return;
         }
         $basePath = $event->getRequest()->getBasePath();
+        $jquery = $this->env == 'prod' ? 'jquery.min.js' : 'jquery.js';
```

Comment:
Whatever is decided, it should never check for `env == 'prod'` but rather `env == 'dev'`, because the Zikula 2.0 user could define own prod-like environments.